BraintreeCore/src/main/java/com/braintreepayments/api/core/BraintreeClient.kt

```diff
@@ -389,4 +389,8 @@ class BraintreeClient @VisibleForTesting internal constructor(
         val message = "Valid authorization required. See $clientSDKSetupURL for more info."
         return BraintreeException(message)
     }
+
+    fun isPayPalInstalled(): Boolean {
+        return deviceInspector.isPayPalInstalled(applicationContext)
+    }
```

Comment:
```suggestion
    var isPayPalInstalled: Boolean
      get() = deviceInspector.isPayPalInstalled(applicationContext)
```
nit: shorthand to make it a property